spec/support/acceptance/nssdb.rb

```diff
@@ -0,0 +1,65 @@
+begin
+  require 'serverspec'
+rescue LoadError
+  # Not using acceptance tests
+else
+  module Serverspec
+    module Type
+      class Nssdb < Command
+        def certificates
+          # TODO: parse the output
+          command_result
```

Comment:
Note to self: this should probably be
```suggestion
          stdout
```